src/js/components/Select/Select.js

```diff
@@ -182,11 +182,23 @@ const Select = forwardRef(
         setValue(nextValue);
         if (onChange) {
           event.persist();
-          const adjustedEvent = event;
-          adjustedEvent.target = inputRef.current;
-          adjustedEvent.value = nextValue;
-          adjustedEvent.option = option;
-          adjustedEvent.selected = nextSelected;
+          let adjustedEvent;
+          // support for native event used by Preact
+          if (event instanceof Event) {
+            adjustedEvent = new event.constructor(event.type, event);
+            Object.defineProperties(adjustedEvent, {
+              target: { value: inputRef.current },
```

Comment:
This looks like `target.value` will be a reference to the input element. Can this be `target: inputRef.current,` instead?

Reply:
You confused me for a moment...
`Object.defineProperties` expects this type of value where we have `{ propertyName: { value: 'someValue', writable: true/false, configurable: true/false, ... } }`. Here's [an MDN example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperties#examples).
And a quick test:
![screenshot](https://user-images.githubusercontent.com/1783133/115811063-0dc39500-a3bd-11eb-8284-8b71d0c78b45.png)

Reply:
Ah, I learned something new today, thanks.